web/src/dynamic-pages/analyze/sections/0-Overview.tsx

```diff
@@ -97,7 +97,7 @@ export const OverviewSection = forwardRef(function (_, ref: ForwardedRef<HTMLEle
           : (
           <>
             <Stack direction='row' flexWrap='wrap' justifyContent='space-between' alignItems='center'>
-              <H1 sx={{ mt: 2 }}>
+              <H1 sx={{ mt: 2 }} noWrap>
```

Comment:
I used the noWrap option in the typography components in material-ui.

https://mui.com/material-ui/api/typography/#:~:text=a%20bottom%20margin.-,noWrap,-bool

If you don't want to show the ellipsis, I would fix it to use `word-break: break-all`.